### ISSUE TITLE
Ember-Bootstrap doesn't seem to be working with Ember 2.12

### ISSUE THREAD

**Issue by mansona** [kaliber5/ember-bootstrap#278]
Hi there, 

We've just updated to Ember 2.12 and we've noticed that none of the ember-bootstrap components are available to the main application. 

The error we are getting is: 

```
Could not find module `ember-bootstrap/components/bs-navbar` imported from `my-app/components/bs-navbar`
```

And loads of other errors for all of the other components.

I am assuming this has something to do with the code in the index.js broccoli file and how it is interacting with the new version https://github.com/kaliber5/ember-bootstrap/blob/master/index.js#L167 or perhaps there is something new in ember-resolver 3.0.0 (but it's hard to tell because Changelog hasn't been updated for ember-resolver)

Don't really know how to start debugging this but let me know if there is anything I can do to exercise this bug 👍

**Comment by simonihmig** [comment]
There was an issue with ember-cli 2.12 (so not related to Ember itself), that was already fixed with the latest `1.0.0-alpha.8`. Seems you are using a previous version, could you please try with the new?

**Comment by mansona** [comment]
Hey @simonihmig, I missed the upgrade because it didn't show up in `npm-check -u`. I suppose it ignores pre-releases ¯\_(ツ)_/¯

I upgraded to alpha.8 and it's working now thanks 🎉
